conf/airframes/ardrone2_optitrack.xml

```diff
@@ -144,11 +129,51 @@
     <define name="DDGAIN_R" value="300"/>
   </section>
 
+  <section name="STABILIZATION_ATTITUDE_INDI" prefix="STABILIZATION_INDI_">
+    <!-- setpoints -->
+    <define name="SP_MAX_PHI" value="60." unit="deg"/>
```

Comment:
These defines will have no effect since they have the wrong prefix (`STABILIZATION_INDI_` instead of `STABILIZAITON_ATTITUDE`).